Libraries/D3D12RaytracingFallback/src/FallbackLayerUnitTests/fallbacklayerunittests.cpp

```diff
@@ -2769,6 +2770,78 @@ void AllocateUAVBuffer(ID3D12Device &d3d12device, UINT64 bufferSize, ID3D12Resou
             }
```

Comment:
Can you also add a similar GeometryFlag check for the LoadAABB test?

Reply:
Am I checking the AABB's metadata in this case? I wasn't sure just because I assumed they didn't have geometry flags.

Reply:
Yes, the metadata applies to both triangles and AABBs. The use of Primitive in PrimitiveMetadata is intentionally used over something like TriangleMetadata to point out that this metadata applies to all primitive types (triangles/AABBS/future other stuff?)